device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/auth/IotHubSasTokenAuthenticationProvider.java

```diff
@@ -35,7 +35,7 @@
     protected IotHubSasToken sasToken;
 
     public abstract boolean canRefreshToken();
-    public abstract String getRenewedSasToken(boolean proactivelyRenew) throws IOException, TransportException;
+    public abstract String getRenewedSasToken(boolean proactivelyRenew, boolean forceRenewal) throws IOException, TransportException;
```

Comment:
current references to this function are always with the same values of forceRenewal and proactivelyRenew.  I see this addition of the argument as unnecessary.  But not a blocker...